packages/faustwp-cli/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@faustwp/cli",
-  "version": "1.1.4",
+  "version": "1.1.3",
```

Comment:
Why is the version being downgraded here?

Reply:
This was probably a bad merge conflict. We will need to get this resolved before approving.

Reply:
Good call out, fixed.